src/components/IconComponent/IconComponent.jsx

```diff
@@ -0,0 +1,12 @@
+import WeatherIcon from "../../img/Rainy.png";
```

Comment:
Let's rename this file into `BackComponent.jsx`

It can be misunderstand what's here